lib/templates.php

```diff
@@ -69,13 +69,12 @@ function gutenberg_register_template_post_type() {
 		'supports'          => array(
 			'title',
 			'slug',
+			'excerpt',
 			'editor',
 			'revisions',
 		),
 	);
 
-	register_post_type( 'wp_template', $args );
```

Comment:
That's surprising duplication, good catch 👍

Reply:
Yes very weird!
Apparently double registrations don't throw any errors? 🤔 

In this same PR I did try "overloading" the `auto-draft` status registration, to add the various missing labels and properties.
Eventually decided against it as I couldn't find a way to limit to just the `wp_template` post type, but otherwise it was working perfectly! 🤷